### PR TITLE
Pol 266 google unlabeled resources

### PR DESCRIPTION
**Pull request by gopinathmahakud1**
### Description

Find all Google cloud resources( disks, images, instances, snapshots, buckets, vpnGateways ) missing any of the user provided labels with the option to update the resources with the missing labels.

### Issues Resolved

- Created a new compliance policy for google cloud. 
- Included README.md, CHANGELOG.md and unlabeled_resources.pt file.
- Updated in root README.md file.
- Added new resource table changes.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5ecfc46df6567200011434f7

image

### CODE REVIEW

File: compliance/google/unlabeled_resources/unlabeled_resources.pt [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+policy "policy_azure_resource" do
+  validate $ds_filtered_resources do
+    summary_template "{{ len data }} Google Resources out of compliance."
+    escalate $esc_email
+    escalate $esc_label_resources
+    check eq(size(data),0) 
+    export do
+      resource_level true
+      field "id" do
+        label "Resources Id"
```
Comment (cdwilhelm) [inline compliance/google/unlabeled_resources/unlabeled_resources.pt:312]: Lets move resource id to the last column.  Make Resource Type the first column, then Resource name.

File: compliance/google/unlabeled_resources/unlabeled_resources.pt [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+escalation "esc_email" do
+   automatic true
+   label "Sends Email"
+   description "Sends Incident Email"
+   email $param_email
+end
+
+escalation "esc_label_resources" do
+   automatic false
+   parameter "param_labels_to_add" do
```
Comment (cdwilhelm) [inline compliance/google/unlabeled_resources/unlabeled_resources.pt:343]: missing label and description

**Comment by cdwilhelm** [comment]
@gopinathmahakud1   this PR needs to be updated with the automatic actions parameter

**Comment by gopinathmahakud1** [comment]
Do we need to add automatic parameter for this policy? I believe always need to be false since, we need to supply parameter while performing add label action.

**Pull request merged**